SPECS/kernel/kernel.spec

```diff
@@ -230,7 +230,7 @@ BuildRequires:  openssl
 BuildRequires:  openssl-devel
 BuildRequires:  pam-devel
 BuildRequires:  procps-ng-devel
-BuildRequires:  python3
+BuildRequires:  python3-devel
```

Comment:
I was wondering that, should we have this `BR` with the `python3-perf` package, as that is one that requires it IMO.
Thoughts ?